src/layout/index.js

```diff
@@ -1,39 +1,81 @@
+import F from 'futil-js'
 import Awaiter from './Awaiter'
+import BarChart from './BarChart'
+import Checkbox from './Checkbox'
+import CheckButton from './CheckButton'
+import DefaultIcon from '../DefaultIcon'
+import Dynamic from './Dynamic'
+import ExpandableTable from './ExpandableTable'
 import Flex from './Flex'
 import Grid from './Grid'
+import LensInput from './LensInput'
 import Modal from './Modal'
-import Popover from './Popover'
-import SpacedList from './SpacedList'
-import TextHighlight from './TextHighlight'
 import ModalPicker from './ModalPicker'
 import NestedPicker from './NestedPicker'
-import Dynamic from './Dynamic'
-import BarChart from './BarChart'
-import { Tag, TagsInput } from './TagsInput'
-import LensInput from './LensInput'
-import ExpandableTable from './ExpandableTable'
+import Popover from './Popover'
 import Portal from './Portal'
-import CheckButton from './CheckButton'
+import RemoveTagIcon from './RemoveTagIcon'
+import SpacedList from './SpacedList'
 import StepsAccordion, { AccordionStep } from './StepsAccordion'
+import { Tag, TagsInput } from './TagsInput'
+import TextHighlight from './TextHighlight'
 
-export {
+import { defaultTheme } from '../utils/theme'
+
+let theme = {
+  AccordionStep,
   Awaiter,
+  BarChart,
+  Button: 'button',
+  Checkbox,
+  CheckButton,
+  Dynamic,
+  ExpandableTable,
   Flex,
   Grid,
+  Icon: DefaultIcon,
+  Input: 'input',
+  Item: 'li',
+  LensInput,
   Modal,
+  ModalPicker,
+  Picker: NestedPicker,
   Popover,
+  PopoverContents: 'div',
+  Portal,
+  RemoveTagIcon,
   SpacedList,
-  TextHighlight,
-  NestedPicker,
-  ModalPicker,
-  Dynamic,
-  BarChart,
+  StepsAccordion,
+  Table: 'table',
   Tag,
+  TagComponent: Tag,
   TagsInput,
-  LensInput,
+  TextHighlight,
+  Wrap: 'i',
+}
+F.mergeOn(defaultTheme, theme)
```

Comment:
Why don't you like `require`ing the base theme here: https://github.com/smartprocure/contexture-react/pull/291/files#diff-e1c6d5d34fa2955dc0df3284a194bce3R27

Reply:
From @michsa
> it still doesn’t work on context initialization, though, so we’d lose providerless defaults

Reply:
I'm not sure that'd be the case. Every themeable component is wrapped in a `ThemeProvider` which merges the base theme so initializing the context is unnecessary.

Reply:
Initializing the context is necessary to support usage without a `ThemeProvider`

Reply:
Ok as per @daedalus28 clarification, we don't want to merge the default theme on `useTheme` every time, so this comment is irrelevant.